bciers/apps/registration1/e2e/poms/operations.ts

```diff
@@ -313,7 +313,6 @@ export class OperationsPOM {
               this.buttonConfirmModal,
               this.alertDeclined,
```

Comment:
This is already checking the proper alert message that is appearing properly now in this PR.